source/refined-github.css

```diff
@@ -1,10 +1,9 @@
 :root {
-	--github-green: #28a745;
-	--github-red: #cb2431;
+	--github-green: var(--color-text-success, #28a745);
```

Comment:
Let's add a comment here mentioning that these variables only exist for GHE and should be removed in a few months.